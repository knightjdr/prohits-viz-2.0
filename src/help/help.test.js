import React from 'react';
import { shallow } from 'enzyme';

import Help from './help';

describe('Help', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(
      <Help />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
