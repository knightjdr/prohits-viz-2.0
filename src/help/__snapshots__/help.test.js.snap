// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Help It renders 1`] = `
<div
  className="Help-container"
>
  <NavbarContainer
    background={true}
    fixed={true}
    links={
      Array [
        Object {
          "route": "/analysis",
          "text": "ANALYSIS",
        },
        Object {
          "route": "/visualization",
          "text": "VISUALIZATION",
        },
        Object {
          "route": "/news",
          "text": "NEWS",
        },
      ]
    }
  />
  <div
    className="Help-content"
  >
    <HelpBarContainer />
    <withRouter() />
  </div>
</div>
`;
